ERCS/erc-7222.md

```diff
@@ -0,0 +1,58 @@
+---
```

Comment:
```suggestion
---
eip: 7608
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename.

Reply:
OK